src/style.css

```
@@ -0,0 +1,135 @@
+.wrapper {
+  position: relative;
+  display: flex;
+  min-width: 100px;
+  margin-top: 20px;
+}
+
+/*Big Bar Default*/
+
+.searchBig {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: 100%;
+}
+
+.search-icon1 {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.searchBig::placeholder {
+  position: absolute;
+  left: 0%;
+  right: 18.35%;
+  top: 0%;
+  bottom: -15.79%;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  /* identical to box height */
+  width: 129px;
+  height: 22px;
+  color: #3d4e61;
+  padding: 25px 709px 23px 62px;
+}
+
+/*Big Bar Hover*/
+
+.searchBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+/*Big Bar Focus*/
+
+.searchBig:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  /* identical to box height */
+  color: #3d4e61;
+  padding: 0 677px 0 62px;
+}
+
+/*Small bar Default*/
+
+.searchSmall {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  width: 100%;
+}
+
+.search-icon2 {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.searchSmall::placeholder {
+  /* position: absolute; */
+  /* left: 0%;
+  right: -6%;
+  top: 0%;
+  bottom: -5.26%; */
+  font-family: "Avenir-Book", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 20px;
+  width: 167.29px;
+  height: 20px;
+  color: #3d4e61;
+  padding: 12px 227px 10px 32px;
+}
+
+/*Small Bar Hover*/
+
+.searchSmall:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+/*Small Bar Focus*/
+
+.searchSmall:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  /* identical to box height */
+  color: #3d4e61;
+  padding: 0 81px 0 33px;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: src/fonts;
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: src/fonts;
+}
```

fonts should be at the top in the css
I am not sure where you are getting these fonts
if you are importing directly from google fonts then should look like this

```suggestion
@font-face {
  font-family: "Avenir-Book";
  src: url("/fonts/Avenir-Book.ttf");
}

@font-face {
  font-family: "Avenir-Heavy";
  src: url("fonts/Avenir-Heavy.ttf");
}
```

---

Hello Amit and thank you for helping me. I fixed the mistakes and the report says "6 passed 0 failed" - lint is also all perfect. Despite all this, when I push my task on Github, looks like my checks are failed and I don't understand why.